docs/src/__examples__/Button/DEFAULT.tsx

```diff
@@ -1,10 +1,45 @@
-import * as React from "react";
+import React from "react";
 import { Button } from "@kiwicom/orbit-components";
 
 export default {
   Example: () => <Button>Click me</Button>,
-  info: {
-    title: "Default button",
-    description: "The default button is a primary button with its children as its text.",
-  },
+  exampleKnobs: [
+    {
+      component: "Button",
+      knobs: [
+        {
+          name: "circled",
+          type: "boolean",
+          defaultValue: false,
+        },
+        {
+          name: "fullWidth",
+          type: "boolean",
+          defaultValue: false,
+        },
+        {
+          name: "disabled",
+          type: "boolean",
+          defaultValue: false,
+        },
+        {
+          name: "loading",
+          type: "boolean",
+          defaultValue: false,
+        },
+        {
+          name: "size",
+          type: "select",
+          defaultValue: "normal",
+          options: ["small", "normal", "large"],
+        },
+        {
+          name: "type",
+          type: "select",
+          defaultValue: "primary",
+          options: ["primary", "secondary", "critical", "primarySubtle", "criticalSubtle", "white"],
+        },
+      ],
+    },
+  ],
 };
```

Comment:
Playground's layout for Button is all over the place. Same for ButtonLink, SocialButton, TextLink and probably some others.

<img width="812" alt="Screen Shot 2021-08-31 at 20 19 11" src="https://user-images.githubusercontent.com/471278/131555539-82540168-9559-4875-bd84-cabea4af1946.png">

Reply:
My bad, now it looks ok to me, so I guess I forgot to build `@kiwicom/orbit-components` or something.